### PR TITLE
Packaging updates

### PR DESCRIPTION
**Pull request by scollazo**
Thie is is a cleaned up version of the package updates made in the past.

To do:
- [ ]  Add documentation
- [ ] Unify build systems / paths / vars

### CODE REVIEW

**Comment by sevein** [comment]
@scollazo, I have some suggestions about the siegfried package:
- Use go 1.9 - tests pass and it's backward compatible. Or even if you decide to stick with go 1.7, I would use go `1.7.6` instead, that's the latest release available.
- Target `cmd/...` when using `go install` to be explicit about where are the entry points that we are trying to compile.
- Stick with siegfried 1.7.3 until we update archivematica-fpr-admin?

I've prepared a patch here: https://gist.github.com/sevein/f27c60678a7db1b0721e026bded8c600.

**Comment by scollazo** [comment]
@sevein I won't update siegfried on the repo until fpr is updated.

Other than that, I want to align the output folder for all packages, in order to tie them to jenkins and manage the repos from there.

**Comment by scollazo** [comment]
Added your patch to the sf rpm. For ubuntu, we use the go packaged through the gophers ppa, and as 1.9 is available, I'll try to align the go version used to build siegried.

**Comment by scollazo** [comment]
@sevein any other concerns for merging this?

File: rpm/fido/Makefile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+NAME          = fido
+VERSION       = 1.3.5
+RELEASE	      = 88
```
Comment (sevein) [inline rpm/fido/Makefile]: Here you're missing tabs and whitespaces. This is happening consistently across many of the files you're changing in this PR. Can you fix it?

**Comment by sevein** [comment]
Hi @scollazo. I'll take a look. Should we have a third person looking at them? I think it'd be worth it but I'm not sure who's the best person. What do you think @jhsimpson?

**Comment by mamedin** [comment]
Here is the third person! I'll take a loot to the rest.

**Comment by sevein** [comment]
@scollazo, does #59 include all the changes done here? If so, can we close this?

**Comment by scollazo** [comment]
Yes, it does, I made different pr's thinking about spliting the code review process between dependencies and archivematica 1.7 packages. Do you think reviewing all changes is easier? If so, feel free to close this.

**Comment by sevein** [comment]
That's a great idea. Why don't you change #59 to target `dev/package-updates`? That way the reviewer can see the differences but you still have the whole set of changes for testing. We've done that in AM.

**Pull request merged**